jax/experimental/ode.py

```diff
@@ -362,13 +368,13 @@ def vjp_all(g, yt, t):
 
     return tuple([result[-4], vjp_times] + list(result[-2]))
 
-  primals_out = odeint(flat_func, y0, t, flat_args)
+  primals_out = odeint(flat_func, y0, t, flat_args, rtol=rtol, atol=atol, mxstep=mxstep)
```

Comment:
a question to reviewers: is it intended to omit `rtol`, `atol` here?

Reply:
@dsweaver any thoughts?

Reply:
Not intended